charts/pulsar/templates/_helpers.tpl

```diff
@@ -84,3 +84,15 @@ Create the match labels.
 app: {{ template "pulsar.name" . }}
 release: {{ .Release.Name }}
 {{- end }}
+
+{{/*
+Create ImagePullSecrets
+*/}}
+{{- define "imagePullSecrets" -}}
```

Comment:
This template name is global. Even though this template is generic, I think we should namespace it with the helm chart's name. It could instead be `pulsar.imagePullSecrets`.

Here is a reference to the helm [documentation](https://helm.sh/docs/chart_template_guide/named_templates/#declaring-and-using-templates-with-define-and-template) for this convention:

> As mentioned above, template names are global. As a result of this, if two templates are declared with the same name the last occurrence will be the one that is used. Since templates in subcharts are compiled together with top-level templates, it is best to name your templates with chart specific names. A popular naming convention is to prefix each defined template with the name of the chart: {{ define "mychart.labels" }}.